app/views/state_file/questions/az_qualifying_organization_contributions/_form.html.erb

```diff
@@ -0,0 +1,71 @@
+<% @main_heading = t('.main_heading', filing_year: @filing_year) %>
+
+<% content_for :page_title, @main_heading %>
+
+<% content_for :card do %>
+  <h1 class="form-question">
+    <%= t('.main_heading_html', filing_year: @filing_year) -%>
+  </h1>
+
+  <%= form_with(
+    model: @contribution,
+    url: {
+      action: @contribution.persisted? ? :update : :create
+    },
+    local: true,
+    builder: VitaMinFormBuilder,
+    html: {
+      class: 'form-card form-card--long'
+    }
+  ) do |f| %>
+
+    <div class="question-with-follow-up">
+      <div class="question-with-follow-up__question">
+        <div class="blue-group">
+          <%=
+            f.cfa_radio_set(
+              :made_contributions,
+              collection: [
+                {
+                  value: :yes,
+                  label: t('general.affirmative'),
+                  input_html: {
+                    "data-follow-up": "#contribution-details",
+                  }
+                },
+                {
+                  value: :no,
+                  label: t('general.negative')
+                }
+              ]
+            )
+          %>
+        </div> 
+
+      </div>
+      <div class="question-with-follow-up__follow-up" id="contribution-details">
+        <div class="blue-group">
+          <p class="form-question spacing-below-25">
+            <%= t('.question_header') %>
+          </p>
+          <div class="form-group-tight">
+            <%= f.cfa_input_field(:charity_name, t('.charity_name'), classes: ["form-width--long"]) %>
+            <%= f.cfa_input_field(:charity_code, t('.charity_code'), classes: ["form-width--long"]) %>
+            <%= f.cfa_input_field(:amount, t('.amount', filing_year: @filing_year), classes: ["form-width--long"]) %>
+            <div class="date-select">
+              <%= f.cfa_date_select(
+                :date_of_contribution,
+                t('.date_of_contribution', filing_year: @filing_year),
+                options: {
+                  start_year: @filing_year,
+                  end_year: @filing_year
+                }
+              ) %>
```

Comment:
minor thing: should we have it default to the filing year so people don't have to click on the one option?

Reply:
Very good idea, will do.

Reply:
Set the filing year. This will effectively set the default to `1/1/#{filing_year}` though. I think that's probably fine. I'll let you decide if it satisfies your request and mark as resolved.